docs/en/latest/installation-guide.md

```diff
@@ -216,6 +216,11 @@ brew services start etcd
 
 ### Configuring APISIX
 
+If you install APISIX via Docker, please exec into the apisix pod first:
+```shell
+kubectl exec -it <name-of-apisix-pod> -n <namespace-running-apisix> -- sh
+```
+
```

Comment:
Can you highlight this as a note? You can use the admonition `:::note`.

Please add spaces around the code blocks (between ```shell and the text). There should be a failing lint check otherwise.

If you install APISIX via Docker, you need to exec into the "apisix" container to run the `apisix` command:

```
command to exec into Docker container from host machine and running the apisix command
```

In Kubernetes, you can do this by running:

```
existing command mentioned in the PR
```

Reply:
I should highlight the entire new addition with the admonition `::note`? 🤔

Reply:
Yes, that should be fine I think. Unless there is a better way.